packages/@aws-cdk/aws-appsync/lib/private/schema.ts

```diff
@@ -0,0 +1,115 @@
+import { AuthorizationType } from '../graphqlapi';
```

Comment:
Can we call this file something like `schema-shape-addition.ts`? The current name really doesn't say anything.